.github/workflows/deploy-to-aks-feast.yaml

```diff
@@ -0,0 +1,143 @@
+name: Create AKS cluster, deploy kubeflow-feast Terraform solution and run UATs
+on:
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: 'Kubernetes version to be used for the AKS cluster. Make sure that the corresponding K8s version is supported by the cloud.'
+        required: false
+      uats_branch:
+        description: 'Branch to run the UATs from e.g. main or track/1.10. By default, this is defined by the dependencies.yaml file.'
+        required: false
+  schedule:
+    - cron: "17 02 * * 1"
+
+jobs:
+  deploy-solution-to-aks:
+    name: Deploy CKF + Feast solution to AKS
+    runs-on: ubuntu-24.04
+    env:
+      AZURE_CORE_OUTPUT: none
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Run YAML to Github Output Action
+        id: yaml-output
+        uses: christian-ci/action-yaml-github-output@v2
+        with:
+          file_path: ".github/dependencies.yaml"
+
+      - name: Update ENV variables from inputs if available
+        run: |
+          K8S_VERSION=${{ inputs.k8s_version || env.K8S_VERSION }}
```

Comment:
Question (non blocking): I wonder if this `short-circuit evaluation` works. I was always thinking the left side of the evaluation needs to have a logical expression like they mention [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#example-matching-an-array-of-strings). But I know we use it also in the different workflow so wonder how it can work haha :)

Reply:
I tried running it with and without inputs and it is working